batoid/parse.py

```diff
@@ -39,9 +42,17 @@ def parse_coordSys(config, coordSys=batoid.CoordSys()):
         shift = [x, y, z]
     elif 'shift' in config:
         shift = config.pop('shift')
-    # Leaving rotation out for the moment...
     if shift != [0.0, 0.0, 0.0]:
         coordSys = coordSys.shiftLocal(shift)
+    # At most one (nonzero) rotation can be included and is applied after the shift.
+    rotXYZ = np.deg2rad([config.pop('rot' + axis, 0.0) for axis in 'XYZ'])
```

Comment:
For the moment I think I'd prefer to use radians directly in the yaml to be consistent with, e.g., the arguments to `ObscRectangle`.

Reply:
Ok, done.